cranelift/frontend/src/switch.rs

```diff
@@ -333,6 +304,30 @@ impl Switch {
     }
 }
 
+fn icmp_imm_u128_eq(bx: &mut FunctionBuilder, x: Value, y: u128) -> Value {
```

Comment:
Sorry to be picky here -- can we pass in an `IntCC` condition and then combine `icmp_imm_u128_{eq,uge}`?

Reply:
Already realized that :)